src/lib/mongoBackend/dbConstants.h

```diff
@@ -111,6 +111,7 @@
 #define CSUB_QS                      "qs"
 #define CSUB_PAYLOAD                 "payload"
 #define CSUB_BLACKLIST               "blacklist"
+#define CSUB_ONLYCHANGED             "onlyChanged"
```

Comment:
Given a new field hasn been introduced in DB, it should be described at https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/database_model.md#csubs-collection. For instance:

> **onlyChanged**: a boolean field that specifies if only attributes that change has to be included in notifications (if `onlyChanged `is equal to true) or not (if `onlyChanged` is equal to false or doesn't exist).

Reply:
fixed in f9887f1